apps/frontend/src/app/Data/Characters/RaidenShogun/index.tsx

```diff
@@ -228,6 +238,10 @@ const dmgFormulas = {
     plunge: burstResolve(dm.burst.plunge),
     plungeLow: burstResolve(dm.burst.plungeLow),
     plungeHigh: burstResolve(dm.burst.plungeHigh),
+    energyGen: prod(
+      subscript(input.total.burstIndex, dm.burst.enerGen),
+      sum(1, a4EnergyRestore_)
```

Comment:
```suggestion
      sum(one, a4EnergyRestore_)
```
You'll also need to update the import

Reply:
Tooltip now showing percentage, as expected.

![image](https://user-images.githubusercontent.com/14800198/234971161-3f6e60f5-539e-4228-bc78-ee9d46797dde.png)